packages/vue-cli-plugin-vuetify/generator/templates/v3/src/shims-vuetify.d.ts

```diff
@@ -0,0 +1,3 @@
+declare module 'vuetify'
+declare module 'vuetify/lib/components'
+declare module 'vuetify/lib/directives'
```

Comment:
Add EOF newline.

Reply:
Yes, it really does.  Thanks for catching that.